README.md

```diff
@@ -105,6 +105,10 @@ The executable of Jenkinsfile Runner allows its invocation with these cli option
 
 ```
  # Usage: jenkinsfile-runner -w [warPath] -p [pluginsDirPath] -f [jenkinsfilePath] [other options]
+ --runHome FILE          : Path to the empty Jenkins Home directory to use for
```

Comment:
Does it have to be empty?

Reply:
That was my assumption since originally always a new temporary folder has been created. My [current implementation](https://github.com/jenkinsci/jenkinsfile-runner/pull/106/files#diff-d88d995828fb47f31944e53ec493d4bcR40) checks if the folder is empty.
I'm not sure what might happen if the folder is not empty and already contains (potentially conflicting) files.